src/org.xtuml.bp.doc/Reference/MASL/xtuml2masl/xtuml2masl.md

```diff
@@ -9,18 +9,21 @@ Syntax
 ------------
 ### Domain Export
 ```
-xtuml2masl [-v | -V] [-e] -i <eclipse project> -d <domain component> [ -o <output directory> ]
+xtuml2masl [-v | -V] [-e] [-xf] [-xl] [-xt] -i <eclipse project> -d <domain component> [ -o <output directory> ]
 ```
    
 ### Package Export
 ```
-xtuml2masl [-v | -V] [-e] -i <eclipse project> -p <project package> [ -o <output directory> ] 
+xtuml2masl [-v | -V] [-e] [-xf] [-xl] [-xt] -i <eclipse project> -d <project package> [ -o <output directory> ]
```

Comment:
why did you change the project export from -p to -d?  Looks to me like -p is still valid (used on lin 1213)

Reply:
copy and paste error